http/client.go

```diff
@@ -479,6 +479,16 @@ func (c *Client) Purge(ctx context.Context) error {
 	return err
 }
 
+func (c *Client) HealthCheck(ctx context.Context) error {
```

Comment:
suggestion: This looks like the CLI would benefit from having this command too.

Reply:
Yes but I didn't want to introduce it in this PR. Planning to add it later. #3708